python/pyspark/pandas/tests/computation/test_compute.py

```diff
@@ -101,16 +101,9 @@ def test_mode(self):
         with self.assertRaises(ValueError):
             psdf.mode(axis=2)
 
-        def f(index, iterator):
```

Comment:
IIRC, this one is by design. Using the specific partitioning to test a mode bug, see https://github.com/apache/spark/pull/38385/files

but I'm fine to remove this one now

Reply:
on second thought, I suggest splitting `test_mode` and then always skip this part

Reply:
Thanks for the context! Just updated the tests.